README.md

```diff
@@ -88,14 +134,3 @@ Database project containing setup in order to the create the QnA Database
 	
 - projects/{subfolder}/sections
 	- Holds QnAData for each WorkflowSection
-	
-_NOTE:_ To publish the database to your local SQL Server, you need to ensure that `ProjectPath` contains the full path.
-- {drive}:\{project-folders}\das-qna-api\src\SFA.DAS.QnA.Database\
-- For example: *C:\projects\efsa\das-qna-api\src\SFA.DAS.QnA.Database\*
```

Comment:
Unless this works out of the box (which I am doubtful of), can you keep this in?

Reply:
Is it still there - Just that I've moved it to the more standard database publish section above